components/use-cases/datahub/datahub.md

```diff
@@ -0,0 +1,31 @@
+---
+title: DataHub
+permalink: /components/use-cases/datahub/
+---
+
+* TOC
+{:toc}
+
+The DataHub use case template provides a one-click setup of an end-to-end flow for populating Keboola metadata 
+into a DataHub data catalog. This includes extracting data from both the Telemetry and Metadata extractors 
+before transforming the data to calculate table properties and lineage. You can choose whether or not to include schemas, 
+properties, or tags from the DataHub writer.
+
+**The flow, in a nutshell:**
+
+![DataHub Flow](/components/use-cases/datahub/datahub-flow.png)
```

Comment:
toto je podle mě fajn tu zobrazit to flow, u advertising-platforms to není a je to dost abstraktní

Reply:
@MonikaFeigler

Reply:
u advertising platforms přidám

Reply:
přidáno